include/circt/Dialect/Seq/SeqPasses.h

```diff
@@ -33,6 +33,7 @@ std::unique_ptr<mlir::Pass> createLowerSeqHLMemPass();
 std::unique_ptr<mlir::Pass>
 createExternalizeClockGatePass(const ExternalizeClockGateOptions &options = {});
 std::unique_ptr<mlir::Pass> createLowerFirMemPass();
+std::unique_ptr<mlir::Pass> createLowerSeqFIFOPass();
```

Comment:
Not for this PR, but at some point we probably want a lowerSeqPass to amortize the walk of the IR over all op types being lowered.